unet/train.py

```diff
@@ -46,7 +52,8 @@ def train_unet(
     model.compile(optimizer="rmsprop", loss="sparse_categorical_crossentropy")
 
     callbacks = [
-        keras.callbacks.ModelCheckpoint("carla_segmentation.h5", save_best_only=True)
+        keras.callbacks.ModelCheckpoint(f"{checkpoint_directory}/unet.h5", save_best_only=True)
+        keras.callbacks.BackupAndRestore(backup_dir="./checkpoints/")
```

Comment:
you can use the `checkpoint_directory` variable that gets passed in for this bit, right?

Reply:
Yeah i probably should. Corrected.